examples/handling/interface/rasterize.py

```diff
@@ -0,0 +1,56 @@
+"""
+Rasterize a vector
+==================
+
+This example demonstrates the rasterizing of a vector using :func:`geoutils.Vector.rasterize`.
+"""
+# sphinx_gallery_thumbnail_number = 2
```

Comment:
This line gets rendered, it probably shouldn't be?

Reply:
Solved in [9d487d0](https://github.com/GlacioHack/geoutils/pull/351/commits/9d487d0a85fe2acffbeb4b75d2e3765d23090f66)